### PR TITLE
Environment variables for e2e tests

### PR DESCRIPTION
**Pull request by GSTVAC**

### CODE REVIEW

**Comment by GSTVAC** [comment]
Hi @sah2ed , thanks for your help. The changes you requested are done.
Let me know if you need something else.

File: e2e/login.spec.js [hunk 1]
```diff
@@ -3,13 +3,21 @@
 describe('The login view', function () {
   var page;
   var mainPage;
+  var testUser = process.env.TEST_USER;
+  var testPassword = process.env.TEST_PASSWORD;
+  var testPort = process.env.TEST_PORT || 3000;
```
Comment (sah2ed) [inline e2e/login.spec.js]: Prior to the start of the tests, there should be a check like:
```
if (testUser == null || testPassword == null) {
// print out warning explaining that tests will fail unless TEST_USER and TEST_PASSWORD environment variables are set.
}

**Comment by sah2ed** [comment]
@GSTVAC 

I ran the build now *without* setting the env variables and it wasn't easily apparent why the tests failed on CircleCI.

It was clearly stated in [Requirement 2.2.1](https://www.topcoder.com/challenge-details/30057220/?type=develop) that *"If either variable is unset, the tests should fail hinting that they must be set for the tests to run."*

Please see my review remark above on how to remedy the issue: https://github.com/topcoder-platform/admin-app/pull/65/files#diff-f1b7e32bde00048fe7a9882a0fbaa700R6

**Comment by GSTVAC** [comment]
@sah2ed 
I agree with you. The message gets lost in the middle of the stack traces.
I decided to throw an exception to make the message more clear. Is it good like this or do you prefer to show just a warning?

**Comment by GSTVAC** [comment]
I submitted another PR with these changes. #66

**Pull request merged**